src/languages/vhdl.coffee

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+
+  name: "Vhdl"
+  namespace: "vhdl"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "VHDL 2008"
+  ]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    "vhd"
+    "VHD"
+  ]
+
+  ###
+
+  ###
+  options:
+    # JavaScript
+    emacs_path:
```

Comment:
This emacs_path option is no longer needed as use of Executables on line 12-20 replaces it.